apps/user/schema.py

```diff
@@ -0,0 +1,60 @@
+from typing import Union
+
+import graphene
+from graphene_django import DjangoObjectType
+from graphene_django_extras import DjangoObjectField, PageGraphqlPagination
+
+from utils.graphene.types import CustomDjangoListObjectType
+from utils.graphene.fields import DjangoPaginatedListObjectField
+
+from .models import User
+from .filters import UserFilterSet
+
+
+class UserType(DjangoObjectType):
+    class Meta:
+        model = User
+        fields = (
+            'id', 'email', 'first_name', 'last_name', 'is_active'
+        )
+
+    display_name = graphene.String()
+
+    @staticmethod
+    def resolve_display_name(root, info, **kwargs):
+        return root.profile.get_display_name()
```

Comment:
Can we use a property decorator and define it somewhere in models?

Reply:
This is defined in the model but in the profile. I am thinking of using the abstract model in deep too (to avoid additional profile join).

Reply:
Using property.